vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher_list_updater.h

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_PUBLISHER_PUBLISHER_LIST_UPDATER_H_
+#define BRAVELEDGER_PUBLISHER_PUBLISHER_LIST_UPDATER_H_
+
+#include <functional>
+#include <map>
+#include <string>
+
+#include "base/timer/timer.h"
+#include "bat/ledger/ledger.h"
+
+namespace bat_ledger {
+class LedgerImpl;
+}
+
+namespace braveledger_publisher {
+
+// Automatically updates the publisher prefix list store on regular
+// intervals.
+class PublisherListUpdater {
```

Comment:
In the future, and likely as a blocker for "rewards everywhere", we will transition to distributing the publisher prefix list using component updater. That effort will require some architectural work in the rewards service.

Reply:
can you please log an issue for it, so that we don't miss it